app/_hub/kong-inc/ai-proxy-advanced/how-to/_load-balancing.md

```diff
@@ -279,4 +283,62 @@ plugins:
       auth: 
         header_name: Authorization
         header_value: Bearer <token>
+```
+
+## Priority
+
+In priority algorithm, targets that have same `weight` are identified as a group. By default, all models have the same priority. However, this can be configured with the [`config.targets[].weight`](/hub/kong-inc/ai-proxy-advanced/configuration/#config-targets-weight) parameter.
```

Comment:
```suggestion
In the priority algorithm, targets that have same `weight` are identified as a group. By default, all models have the same priority. However, this can be configured with the [`config.targets[].weight`](/hub/kong-inc/ai-proxy-advanced/configuration/#config-targets-weight) parameter.
```